lib/aussieaddonscommon/utils.py

```diff
@@ -218,18 +218,49 @@ def log_kodi_platform_version():
     log("Kodi %s running on %s" % (version, platform))
 
 
+
+def check_country(connection_info, message=None):
```

Comment:
this function returns a bool, so would be nice to have a clear method name here like is_valid_country or valid_country?